File: scripts/Detect Capital/Detect Capital.java
class Solution {
    public boolean detectCapitalUse(String word) {
        int count = 0;
        for(int i=0; i < word.length(); i++){
            if('A' <= word.charAt(i) && word.charAt(i) <= 'Z')
                count++;
        }
        if(count == 0 || count == word.length() || (count == 1 && ('A' <= word.charAt(0) && word.charAt(0) <= 'Z')))
            return true;
        else
            return false;
    }
}

File: algorithms/D/Detect Capital/Detect Capital.js
var detectCapitalUse = function(word) {
      if((word.charAt(0)==word.charAt(0).toUpperCase() && word.slice(1)==word.slice(1).toLowerCase()) || (word == word.toUpperCase() 
   || word == word.toLowerCase()))
   {
      return true
   }
    else{
      return false
  }
};
